android/app/src/main/res/values/strings.xml

```diff
@@ -170,5 +170,14 @@
     <string name="whatsapp_promo_msg_template">Hey, join me at the event! - %s</string>
     <string name="error_msg_retry">Please Retry Again</string>
     <string name="app_share_url">https://play.google.com/store/apps/details?id=%s</string>
+    <string name="share_app">Share App</string>
+    <string name="invite_via_facebook">Invite Via Facebook</string>
+    <string name="facebook_app_id">1786614831603417</string>
+    <string name="share_fb_preview_url">https://lh3.googleusercontent.com/ai5OksP-YP6DdCuqCxE-97Kz9lEFu7VtDVnhfXB__h6K0gx3Rrl0mWM9mXsfMb19pqw=w300-rw</string>
+
+    <!--Branch integration strings-->
+    <string name="branch_key_debug">key_test_gjCjmUMS5Dp6u7IukMSDujjhDwbPyjbB</string>
+    <string name="branch_key_live">key_live_pavjjGHL2Al0qWQzjRQFvcddwscRvkcp</string>
+    <string name="app_link_url">https://fossasia.app.link/rwWTYZ0tLA</string>
```

Comment:
About this, can we make this a dynamic link rather than making it a static url.
Because at the end, this repo is just a template and has to be used by other event organizers too for creating their own app.